packages/container/lib/container.js

```diff
@@ -15,7 +17,8 @@ import dictionary from 'ember-metal/dictionary';
  @private
  @class Container
  */
-function Container(registry, options) {
+function Container(owner, registry, options) {
```

Comment:
first arg means existing Container usage will break, any downside to this being in `options` or a third arg?